### ISSUE TITLE
Access Denied

### ISSUE THREAD

**Issue by Soupy710** [bluejeans/android-client-sdk#3]
I'm getting the below access denied error on building an android app with this SDK.

Could not resolve com.bluejeans:android-client-sdk:1.7.0.
         > Could not get resource 's3://risk-manager-android-sdk/artifacts/com/bluejeans/android-client-sdk/1.7.0/android-client-sdk-1.7.0.pom'.
            > Access Denied (Service: Amazon S3; Status Code: 403; Error Code: AccessDenied; Request ID: 03HEEBK5B6QH205E; S3 Extended Request ID: tRMwsktBtUGYJghionZGu9qYFLdE3AjauSRcT0uuZI6+ufitLlXZaQKnZF3IVk4qbkYeEsf2VWw=; Proxy: null)

**Comment by pmathew92** [comment]
Hi @Soupy710 ,
 Did you add  `repositories { maven { url "https://swdl.bluejeans.com/bjnvideosdk/android" } }`
dependency in your projects top-level build.gradle file?

Could you try downloading our sample project and build them ? If you are able to build  them, please check if you are missing any required dependency or not from the sample project

**Comment by Soupy710** [comment]
Yep, I added that. The sample project runs as expected. I'm not sure if this is a dependency related issue
